cupy/cuda/memory.pyx

```diff
@@ -341,6 +341,12 @@ cdef class SingleDeviceMemoryPool:
     cpdef free_all_free(self):
         self._free = collections.defaultdict(list)
 
+    cpdef n_free_blocks(self):
+        cdef Py_ssize_t n = 0
+        for v in self._free.itervalues():
```

Comment:
Sorry, I made a mistake.
Please use `six.itervalues`, because `itervalues` was removed in Python3.